policy/policy.go

```diff
@@ -148,66 +151,79 @@ func (pr ByResolutionAsc) Less(i, j int) bool {
 	return pr[i].Resolution().Precision < pr[i].Resolution().Precision
 }
 
-// VersionedPolicies represent a list of policies at a specified version.
-type VersionedPolicies struct {
-	// Version is the version of the policies.
-	Version int
-
+// StagedPolicies represent a list of policies at a specified version.
+type StagedPolicies struct {
 	// Cutover is when the policies take effect.
-	Cutover time.Time
+	CutoverNs int64
 
-	// isDefault determines whether the policies are the default policies.
-	isDefault bool
+	// Tombstoned determines whether the associated (rollup) metric has been tombstoned.
+	Tombstoned bool
 
 	// policies represent the list of policies.
 	policies []Policy
 }
 
-// IsDefault determines whether the policies are the default policies.
-func (vp VersionedPolicies) IsDefault() bool { return vp.isDefault }
+// NewStagedPolicies create a new staged policies.
+func NewStagedPolicies(cutoverNs int64, tombstoned bool, policies []Policy) StagedPolicies {
```

Comment:
Can we make this constructor take a default policies and remove the module level `defaultPolicies`?